docker/ci/Dockerfile

```diff
@@ -39,6 +39,15 @@ RUN curl -s https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add
     echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list && \
     apt-get update && apt-get install -y google-chrome-stable
 
+# Install latest kubectl.
+RUN curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl" \
```

Comment:
@oliverchang is this risky? I guess it's only in CI.

Reply:
We should install this via google cloud SDK: https://cloud.google.com/kubernetes-engine/docs/how-to/cluster-access-for-kubectl# 

We already have the Cloud SDK APT repo set up.

Per the "apt" instructions on https://cloud.google.com/kubernetes-engine/docs/how-to/cluster-access-for-kubectl#apt, we might be able to jsut do `apt-get install -y kubectl`?